pkg/pillar/cmd/baseosmgr/handlebaseos.go

```diff
@@ -966,9 +966,9 @@ func baseOsSetPartitionInfoInStatus(ctx *baseOsMgrContext, status *types.BaseOsS
 	if partStatus != nil {
 		log.Infof("baseOsSetPartitionInfoInStatus(%s) %s found %+v\n",
 			status.Key(), partName, partStatus)
-		status.PartitionLabel = partName
-		status.PartitionState = partStatus.PartitionState
-		status.PartitionDevice = partStatus.PartitionDevname
+		status.PartitionLabel = strings.TrimSpace(partName)
```

Comment:
Why did you need to add these here? Any why not add them in 10 other places?
If there is some input which can have spaces we should do a trimpspace where we use that input to set PartionState etc, not in the middle of the code.
If you want to assert that there is no leading or trailing space when you use it that would be fine, but not to actually remove them.

Reply:
sure, will remove these.